recipes/faiss/build-cpu.sh

```diff
@@ -0,0 +1,10 @@
+# Build vanilla version (no avx)
+./configure --without-cuda
```

Comment:
FWIW we can also build the GPU bits on Linux. Happy to advise on how we would do this 🙂

Reply:
I brought this up in the OP:

> 
I'm currently not packaging the gpu version, as there are various unresolved [discussions](https://github.com/conda-forge/pytorch-cpu-feedstock/issues/7) along those lines (even though it seems conda-forge [might](https://github.com/conda-forge/pytorch-cpu-feedstock/issues/7#issuecomment-562220076) package those eventually). The cpu-version should be fine IMO, and it looks like it might be packagable for windows too.

I thought this might likely need a bit more discussion. Otherwise, it should be relatively straightforward.

Reply:
That's more specific to `pytorch-cpu`. Generally it is possible to build the GPU bits of packages. ( https://github.com/conda-forge/ucx-split-feedstock/pull/14 )